src/Layer/Layer.js

```diff
@@ -74,12 +75,6 @@ class Layer extends THREE.EventDispatcher {
             };
         }
 
-        // TODO remove this warning and fallback after the release following v2.3.0
```

Comment:
Have you add breaking change on your message commit?

Reply:
No breaking change yet as there is a deprecation warning message and we still support the current way of adding layers.

Reply:
Oh sorry I misunderstood, yes I'll add a breaking change in my message.